lib/urlwatch/jobs.py

```diff
@@ -230,3 +231,29 @@ def retrieve(self, job_state):
                 return response.content.decode('ascii', 'ignore')
 
         return response.text
+
+class FileJob(Job):
+    """Read the content of a local file"""
+
+    __kind__ = 'file'
+    __required__ = ('path',)
+
+    def get_location(self):
+        return str(self.path)
+
+    def retrieve(self, job_state):
+
+        path_string = str(self.path)
+
+        if not path_string:
+            raise ValueError('Path cannot be null or empty')
+
+        if path_string.startswith('file://'):
```

Comment:
I wouldn't handle this here. Come to think of it, are local files not already supported by using a `file://` URL? If not, maybe they should be? This way, we don't need a new kind of job, but can just use a URL job with `file://`, something that might be more straightforward?